apps/www/src/tests/factories/feature.factory.ts

```diff
@@ -1,15 +1,26 @@
 import {Feature} from '@/models/feature.model'
 import {faker} from '@faker-js/faker'
 import {iconNames, iconTypes} from '@/models/icon.model'
+import {imageFactory} from './image.factory'
 
 export type FeatureFactoryProps = {
-  type: (typeof iconTypes)[number]
+  useImage?: boolean
+  type?: (typeof iconTypes)[number]
 }
 
-export function featureFactory({type}: FeatureFactoryProps): Feature {
+export function featureFactory({type, useImage = false}: FeatureFactoryProps): Feature {
+  if (useImage) {
```

Comment:
Instead of having two objects create a base and add either the icon or the image to it.